public/js/user.js

```diff
@@ -66,10 +66,16 @@ apos.define('apostrophe-workflow', {
         if (err) {
           return;
         }
+
+        // The submit procedure should only affect edited docs, it shouldn't
+        // consider documents that have been trashed (even if they technically
+        // haven't been submitted)
```

Comment:
Why not though? Many users lack permissions to commit. They need a way to submit the stuff they trashed for someone to approve that.

Reply:
My reasoning here was that if we include trashed pages in the submission, then *all* trashed pages will be included, and that would make it difficult to notice which doc needs to be committed. In contrast, for edited pages – only the current page and related pages are included in the submission.

But perhaps it's the better option to include every trashed page in the submission, if the most important thing is to get the attention of a user with commit permission..? I guess we leave users to manually submit pages the same way they have to manually submit/commit today if we don't include trashed pages in the submission.

Reply:
Yes, upon reflection I see no reason why trashed docs shouldn't auto-submit at the time of trashing, because it is unlikely that the user will go back and submit them, and if their other work is approved without those changes they will get unexpected results if joins still exist.

Reply:
... Actually, since the commit button appears if there is anything in the trash that you are allowed to commit (or that any rate will be the case after you make the tweaks I mentioned), we don't need a special effort to submit pieces/pages in the trash. The person with commit privileges will see a commit button as soon as anything goes into the trash in draft but not in live.

Reply:
I think that makes sense. As you say, anybody with commit permissions will see the commit button as soon as something has been trashed, and since the submit functionality is pretty barebones – we don't gain much from including additional pages (email notifications or similar)